test/Main.hs

```diff
@@ -15,4 +16,12 @@ test_all =
         ]
 
 main :: IO ()
-main = defaultMain test_all
+main =
+    do  test_rawParsing <- Raw.gen_test_parsing
```

Comment:
Why not alter `test_all` above (and change its type to `IO TestTree`) instead of inlining everything here?